docs/issue-1695-fix.md

```diff
@@ -0,0 +1,217 @@
+# Fix for Issue #1695: Cyclic Eviction of Large Pods in LowNodeUtilization
```

Comment:
What is the reason for this file ? I haven't seen any other issue creating these.